.github/docs/release.md

```diff
@@ -126,21 +126,21 @@ This checklist will prepare `v1.3.0` from `v1.2.0`. Adjust your version numbers
     * Upload both image and state into the newly created folder.
     * Delete the disk in Azure!
 
-7. To bring updated version numbers and other changes (if any) to main, create a new branch `feat/release` from `release/v1.3`, rebase it onto main, and create a PR to main
-8. Milestones management
+8. To bring updated version numbers and other changes (if any) to main, create a new branch `feat/release` from `release/v1.3`, rebase it onto main, and create a PR to main
+9. Milestones management
    1. Create a new milestone for the next release
    2. Add the next release manager and an approximate release date to the milestone description
    3. Close the milestone for the release
    4. Move open issues and PRs from closed milestone to next milestone
-9. If the release is a minor version release, create an empty commit on main and tag it as the start of the next pre-release phase.
+10. If the release is a minor version release, tag the latest commit on main as the start of the next pre-release phase.
 
     ```sh
-    nextMinorVer=$(echo $ver | awk -F. -v OFS=. '{$2 += 1 ; print}')
+    nextMinorVer=$(echo "${ver}" | awk -F. -v OFS=. '{$2 += 1 ; print}')
     git checkout main
-    git pull
-    git commit --allow-empty -m "Start v$nextMinorVer-pre"
-    git tag v$nextMinorVer-pre
-    git push origin main v$nextMinorVer-pre
+    git pull{nextMinorVer}-pre"
+    git tag v${nextMinorVer}-pre
+    git push
```

Comment:
Is the second push necessary?

Reply:
Ah good catch. Now that we avoided the additional commit, that push can be removed too!